app.rb

```diff
@@ -228,7 +228,11 @@ def events
 }.freeze
 
 get '/' do
-  redirect('/board/hoxton')
+  redirect('/board/dxw')
+end
+
+get '/hoxton' do
+  redirect('board/hoxton')
```

Comment:
Does this need a leading `/` for the redirect?

```suggestion
  redirect('/board/hoxton')
```

Reply:
Interestingly enough it works on my machine, but I'll add it for consistency (and to account for unpredictability of production envs :laughcry:)